src/components/Block/Space.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div style="position: fixed; width: 240px">
+    <Block :slim="true" class="overflow-hidden">
+      <div class="text-center border-bottom header-bg">
+        <Token :space="space" symbolIndex="space" size="80" class="mt-3 mb-2" />
+        <h3 class="mb-3">{{ space.name }}</h3>
+      </div>
+      <div class="py-3">
+        <router-link
+          :to="{ name: 'proposals' }"
+          v-text="$t('Proposals')"
+          :class="$route.name === 'proposals' && 'router-link-exact-active'"
+          class="d-block px-4 py-2 sidenav-item"
+        />
+        <router-link
+          :to="{ name: 'create' }"
+          v-text="$t('proposals.new')"
+          class="d-block px-4 py-2 sidenav-item"
+        />
+        <router-link
+          v-if="
+            space.strategies.find(strategy => strategy.name === 'delegation')
+          "
+          :to="{ name: 'delegate', params: { key: space.key } }"
+          v-text="$t('delegate.header')"
+          class="d-block px-4 py-2 sidenav-item"
+        />
+        <router-link
+          :to="{ name: 'about' }"
+          v-text="$t('about')"
+          :class="$route.name === 'about' && 'router-link-exact-active'"
+          class="d-block px-4 py-2 sidenav-item"
+        />
+        <router-link
+          v-if="isAdmin"
```

Comment:
Not related to these pr changes, but can we also add isOwner here? I have no admins to my space and I can't see this 😋

Reply:
Query for a space owner take too long to resolve, i'd prefer not adding it.